### DISCUSSION TITLE
How can I post marp slide to blogger(blogspot) properly?

### DISCUSSION THREAD

**Discussion by luceat-lux-vestra** [marp-team/marp#173] [Q&A] [answered]
I tested this on my [blog](https://heathkim.blogspot.com/2021/06/vs-code-marp.html?m=1).

I can't control size of presentation.  

Always very small.  

How can I deal with this?  
Thanks.

**Comment by yhatt** [comment, marked as answer]
**Don't use HTML generated by Marp CLI / Marp for VS Code as an embed snippet.** _It is designed for a standalone HTML file to present the deck, not for a snippet to embed to the blog article._ If just copied exported HTML and pasted to blog, your site will be broken surely.

In your situation, recommend to use HTML and CSS snippets generated from [Marp Core](https://github.com/marp-team/marp-core) or [Marpit](https://marpit.marp.app/). If you really want to use HTML file created by [Marp CLI](https://github.com/marp-team/marp-cli) / [Marp for VS Code](https://github.com/marp-team/marp-vscode), consider to use `<iframe>` tag to put the exported HTML. It can control the slide size by `width` and `height` attribtues.

```html
 <iframe src="https://example.com/url-to-your-marp-slide.html" width="720" height="405" />
```

When you're using Marp Core (or Marpit with inline SVG mode), should set a style of inline SVG elements to control the size of slides. [See "Pixel-perfect scaling" section of Marpit documentation](https://marpit.marp.app/inline-svg?id=pixel-perfect-scaling):

```css
svg[data-marpit-svg] {
  display: block;
  width: 720px;
  height: 405px;
}
```

**Reply by mrchantey** [reply]
By the way the iframe didn't work for me without an explicit closing tag:

```html
bad
<iframe src="https://example.com"/>

good
<iframe src="https://example.com"></iframe>

```
